content/unexpected-reboots.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Reboots (Unexpected)
+description: >
+   If your system is spontaneously rebooting, this article provides steps to isolate the cause.
+keywords:
+  - reboot
+  - shutdown
+  - restart
+  - reset
+  - crash
+  - reinstall
+  - refresh
+  - power-off
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: hardware-troubleshooting
+tableOfContents: true
+---
+
+If your system is spontaneously rebooting, this article provides steps to isolate the cause.
+
+> **NOTE:** There is a subtle but important difference between a reboot and a shutdown/power-off. In a reboot, parts of the system are left powered on to facilitate restarting the system. In a shutdown/power-off, hardware from all parts of the system except charging circuitry and capacitors is powered down.
+
+## Troubleshooting (Device Agnostic)
+
+### Software Causes
+
+Anything not directly affected by hardware, is controlled by software. This includes the kernel, drivers, and the desktop environment. The Operating System (OS), is essentially a software program that controls other software programs.
```

Comment:
The comma in the first sentence isn't needed, it should be "Anything not directly affected by hardware is controlled by software."

"This includes the kernel, drivers, and the desktop environment." - I would either add "the" before "drivers" or remove "the" from before "desktop environment" so the three list items are consistent with each other.